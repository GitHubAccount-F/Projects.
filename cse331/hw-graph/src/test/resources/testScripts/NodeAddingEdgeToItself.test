#Tests to see if a node can have an edge
#incoming and outgoing from itself. Also
#tests to make sure this applies to nodes connected
#to other nodes.

#Creates empty graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3
CreateGraph graph4

#Adds a node
AddNode graph1 one
AddNode graph2 one
AddNode graph3 one
AddNode graph4 one
AddNode graph4 two

#Creates an edge
AddEdge graph2 one one e1
AddEdge graph3 one one e1
AddEdge graph3 one one e2
AddEdge graph4 one one e1
AddEdge graph4 one two e1

#Checks to see the child node
ListChildren graph1 one
ListChildren graph2 one
ListChildren graph3 one
ListChildren graph4 one