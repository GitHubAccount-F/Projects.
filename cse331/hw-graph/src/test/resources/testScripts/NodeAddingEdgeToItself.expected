#Tests to see if a node can have an edge
#incoming and outgoing from itself. Also
#tests to make sure this applies to nodes connected
#to other nodes.

#Creates empty graphs
created graph graph1
created graph graph2
created graph graph3
created graph graph4

#Adds a node
added node one to graph1
added node one to graph2
added node one to graph3
added node one to graph4
added node two to graph4

#Creates an edge
added edge e1 from one to one in graph2
added edge e1 from one to one in graph3
added edge e2 from one to one in graph3
added edge e1 from one to one in graph4
added edge e1 from one to two in graph4

#Checks to see the child node
the children of one in graph1 are:
the children of one in graph2 are: one(e1)
the children of one in graph3 are: one(e1) one(e2)
the children of one in graph4 are: one(e1) two(e1)